docs/reference/nodes/workflow.md

```diff
@@ -0,0 +1,44 @@
+# Workflow Nodes
+

```

Comment:
Can we add a brief text here to explain context of what workflow nodes are used for?

Reply:
Yes! I completely missed that, thanks